pkg/controllers/observability/tempo_components.go

```diff
@@ -0,0 +1,300 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+
+	tempov1alpha1 "github.com/grafana/tempo-operator/api/tempo/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	obsv1alpha1 "github.com/rhobs/observability-operator/pkg/apis/observability/v1alpha1"
+	uiv1alpha1 "github.com/rhobs/observability-operator/pkg/apis/uiplugin/v1alpha1"
+)
+
+const (
+	tenantName = "prod"
+	tenantID   = "1610b0c3-c509-4592-a256-a1871353dbfb"
```

Comment:
I guess this should be provided by the user or?

Reply:
I have clarified this in the CR.